extensions/dacpac/src/wizard/api/dacFxConfigPage.ts

```diff
@@ -149,6 +150,7 @@ export abstract class DacFxConfigPage extends BasePage {
 				required: true
 			}).component();
 
+		this.fileTextBox.ariaLabel = localize('dacfx.fileLocationEdit', "File Location Edit");
 		this.fileButton = this.view.modelBuilder.button().withProperties({
 			label: '•••',
```

Comment:
Separate, but this button probably needs an ariaLabel too

Reply:
yup there's an issue for that #6742. Will fix in a separate PR